messages/RunOutputProcessor.js

```diff
@@ -1,7 +1,6 @@
 module.exports = {
 	"output": {
 		"engineSummaryTemplate": "Executed %s, found %s violation(s) across %s file(s).",
-		"noViolationsDetected": "Executed engines: %s. No rule violations found.",
```

Comment:
This message is now no longer used, anywhere. Instead, the above message is used. If there are no violations, a message similar to `Executed sfge, found 0 violation(s) across 0 file(s)` is printed.